pheno001/participants.json

```diff
@@ -0,0 +1,37 @@
+{
+  "participant_id": {
+    "Description": "OpenNeuro ID of the subject."
+  },
+  "sex": {
+    "Description": "Sex assigned at birth.",
+    "Levels": {
+      "m": "Male",
+      "f": "Female"
+    }
+  },
+  "age_at_visit": {
```

Comment:
```suggestion
  "age": {
```

Reply:
In this case, `age_at_visit` would be the same as `age` and that's the reason I changed it to `age_at_visit`.

Reply:
I think it's still fine to keep as `age`, but I would let @SamGuay and/or @surchs be the tie-breakers.